packages/frontend-app-api/src/wiring/createApp.tsx

```diff
@@ -57,9 +57,14 @@ import {
   ApiFactoryRegistry,
   ApiProvider,
   ApiResolver,
-  AppThemeSelector,
 } from '@backstage/core-app-api';
 
+import {
+  AppLanguageSelector,
+  AppThemeSelector,
+  I18nextTranslationApi,
+} from '@backstage/core-app-api/alpha';
```

Comment:
Let's use internal relative imports for now like the things below, don't really want to export these at this point.